packages/x-password-bundle/DOCUMENTATION.md

```diff
@@ -142,6 +142,26 @@ export interface IXPasswordBundleConfig {
 }
 ```
 
+you can also get REST apis versions of those mutation in addition to me Query, if you toggled on/off the rest attribute :
+
+```ts
+// The configuration is:
+export interface IXPasswordBundleConfig {
```

Comment:
For the REST APIs, don't we need documentation of props and their types? Are we using a POST JSON body?